brambling/models.py

```diff
@@ -1098,6 +1098,10 @@ def get_eventhousing(self):
     def has_dwolla_payments(self):
         return self.transactions.filter(method=Transaction.DWOLLA).exists()
 
+    def awaiting_check_confirmation(self):
```

Comment:
Yeah... it would be great if this could get fixed, since it adds one or two queries per event on the dashboard. It should be possible to add a qs of unconfirmed check payments to each event on the dashboard using a [custom Prefetch object](https://docs.djangoproject.com/en/1.9/ref/models/querysets/#django.db.models.Prefetch). Ideally you'd add a count or an exists rather than a full qs, but I don't think that's an option...